libs/go/zmscli/cli.go

```diff
@@ -942,8 +957,8 @@ func (cli Zms) HelpSpecificCommand(interactive bool, cmd string) string {
 	switch cmd {
 	case "list-domain":
 		buf.WriteString(" syntax:\n")
-		buf.WriteString("   list-domain [prefix]\n")
-		buf.WriteString("   list-domain [limit skip prefix depth]\n")
+		buf.WriteString("   [-o json] list-domain [prefix]\n")
```

Comment:
@jothi-prasad I noticed you open the issue for domain commands only so the flag is implemented only for domain-related actions. Any reason why the change isn't for all methods?